File: scharp/day/five/SeleniumTestProject/tests/WebFormTest.cs
using NUnit.Framework;
using SeleniumTestProject.pages;
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace SeleniumTestProject.tests
{
    
    public class WebFormTest : BaseTest
    {

        [Test]
        [DisplayName("User can send the form")]
        public void UserCanSendFormTest()
        {
           SubmittedPage submittedPage = new WebFormPage(Driver, 5)
                .Open()
                .TypeText("Hi, everyone!")
                .Submit();
            Assert.That(submittedPage.MessageText(), Is.EqualTo("Received!"));
        }
    }
}

File: java/java-practise/src/main/java/day/five/selenium-page-object/src/test/java/tests/WebFormTest.java
package tests;

import org.junit.jupiter.api.DisplayName;
import org.junit.jupiter.api.Test;
import pages.SubmittedPage;
import pages.WebFormPage;

import static org.junit.jupiter.api.Assertions.assertEquals;

public class WebFormTest extends BaseTest{

    @Test
    @DisplayName("User can send the form")
    void userCanSendFormTest(){
        SubmittedPage submittedPage = new WebFormPage(driver, 5)
                .open()
                .typeText("Hello")
                .submit();

        assertEquals("Received!", submittedPage.getMessage());
    }
}
